src/js/themes/base.js

```diff
@@ -783,8 +782,6 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         align: 'start',
         pad: { horizontal: 'small', vertical: 'xsmall' },
         border: 'bottom',
```

Comment:
A couple of lines below were removed (with styling of `fill` and `verticalAlign`). are we still allowing backward compatibility with this removals?

Reply:
Good catch! We don't need `fill` anymore, since we aren't using Box as much. But, we did need `verticalAlign`. I've restored that. I also uncovered that this pull request fixes an alignment issue with the current master branch version. :)